src/Controls/FloatingMenu/FloatingActionMenu.xaml.cs

```diff
@@ -241,6 +241,14 @@ public double Size
             declaringType: typeof(FloatingActionMenu),
             defaultValue: default(Thickness));
 
+        public static readonly BindableProperty MainButtonToItemMarginProperty = BindableProperty.Create(nameof(MainButtonToItemMargin), typeof(Thickness), typeof(FloatingActionMenu), default(Thickness));
```

Comment:
Je ne te demande pas un hack. :'(
Je demande que tu corriges le contrôle.
Par défaut il ne doit pas avoir cette marge corrige le layout.

Reply:
C'est noté, je me suis dit que cette approche sera plus flexible, mais visiblement j'ai fait ce qui n'était pas demandé. Je vais remédier à ça.

Reply:
En fait l'idée c'est d'avoir un comportement par défaut sans marge. 
Tu peux laisser ta propriété MainButtonToItemMarginProperty l'idée est bonne.
Mais c'est bizarre de devoir définir -33 pixel marge haute pour avoir le rendu de base.
Désolé si j'ai été un peu sec dans le précédent message. A mercredi !

Reply:
Ok, C'est noté,